step-ca/provisioners.mdx

```diff
@@ -985,32 +985,63 @@ see the [claims](configuration.mdx#claims) section for all the options.
 
-- **attestationFormats**<Reference id="star6" marker="*" />: which attestation formats are allowed for `device-attest-01` challenges. Valid values are `apple`, `step`, and `tpm`. By default, all formats are enabled. The `apple` format is for Apple devices, and adds trust for Apple's CAs. The `step` format is for non-TPM devices that can issue attestation certificates, such as YubiKey PIV. It adds trust for Yubico's root CA. The `tpm` format is for TPMs. It does not trust any TPM vendor CAs by default.
+- **attestationFormats**<Reference id="star6" marker="*" />: which attestation formats are allowed for `device-attest-01` challenges. Valid values are `apple`, `step`, and `tpm`. By default, all formats are enabled. 
 
-- **attestationRoots**<Reference id="star6" marker="*" />: contains a trust bundle of root certificates in PEM format that will be used to verify attestation certificates. The default value depends on the value of **attestationFormats**. If provided, this PEM bundle will override the CA trust established by setting **attestationFormats**.
+- **attestationRoots**<Reference id="star6" marker="*" />: a CA trust bundle that will be used to verify attestation certificates. A base64-encoded string is expected. To generate this string, create a single PEM bundle containing all trusted CA certificates for the provisioner, then base64 encode the bundle. The default value of **attestationRoots** depends on the value of **attestationFormats**. If provided, this PEM bundle will override the CA default trust roots established by setting **attestationFormats**.
 
 <Footnote id="star6" marker="*">Optional</Footnote>
 
-<br /><br />
+<br />
 
 See [ACME Basics](./acme-basics.mdx)
 for more guidance on configuring and using the ACME protocol with `step-ca`.
 
 #### ACME for Device Attestation
 
+The `device-attest-01` challenge supports device attestations from iOS, iPadOS, tvOS, and YubiKeys.
+An attestation certificate is used
+to satisfy the ACME challenge.
+The CSR may include
+a device ID in a `permanentIdentifier` SAN ([RFC 4043](https://www.rfc-editor.org/rfc/rfc4043.html)),
+depending on the application. 
+
+This challenge type is not enabled by default and must be explicitly enabled.
+
+##### Attestation statement formats
+
+The `device-attest-01` challenge response
+uses a variant of the WebAuthn attestation statement format
+adapted for ACME.
+The variations are specified in
+[IETF draft-acme-device-attest](https://datatracker.ietf.org/doc/draft-acme-device-attest/).
+
+`step-ca` supports device attestation statements
+in `tpm`, `apple`, and `step` formats.
+
+* The `tpm` format is a variant of the [WebAuthn TPM Attestation statement](https://www.w3.org/TR/webauthn-2/#sctn-tpm-attestation). By default, it does not add trust for any TPM vendor CAs. You must add TPM vendor trust roots via the `attestationRoots` configuration.
```

Comment:
The attestation roots to be configured in the TPM provisioner are not the TPM manufacturer roots, but those of the attestation CA(s) to trust. The TPM manufacturer roots are used by the attestation CA.